### PR TITLE
Realtime line updation

### PR DESCRIPTION
**Pull request by nick9822**
Hi Team,
This is with reference to #1153 

Please advice whether is it a good way to achieve so I can apply same changes to Sales.

Thanks,
Nick

### CODE REVIEW

**Comment by RamkrishnaMondal** [comment]
WOW! this is very simple yet very effective solution.
Thank you very much. 
Hope @jekkos & @daN4cat  will love this too.

File: application/views/receivings/receiving.php [hunk 1]
```diff
@@ -516,6 +516,17 @@
 			}
 		}
 	}
+	$('[name="price"]').focusout( function(e) {
+		document.getElementById(this.id).submit();
```
Comment (jekkos) [inline application/views/receivings/receiving.php]: Why not use `$(this).submit()` instead of combining jQuery with regular JS?

Comment (jekkos) [inline application/views/receivings/receiving.php]: Also the selectors can be combined which would reduce the change to only three lines, as follows

`$('[name="price"],[name="quantity"]).focusout($(this).submit)`

**Comment by jekkos** [comment]
Also you are duplicating id's among elements in the DOM which isn't allowed in any way. They should be unique at all times

File: application/views/receivings/receiving.php (hunk 1 above)
Comment (nick9822) [inline application/views/receivings/receiving.php]: Nice review. I will check and update pull request.

**Comment by nick9822** [comment]
@jekkos
Yes its not permitted. As classes and names were occupied, I thought to give it a try.

Anyways latest changes will not require ids, so I will remove those.

**Comment by jekkos** [comment]
Thanks for making the PR, I've added this bit in the code myself without using the input field id's. Too bad it does not have you're name on it

**Comment by nick9822** [comment]
@jekkos 
Its okay that I do not have name on it. I am happy that the new feature added to the pos. 🖒😊

OSPOS is good POS, lets build it more perfect. ☺

**Pull request closed without merging**